kubernetes-ingress-defaultbackend.yaml

```diff
@@ -52,7 +52,7 @@ test:
 
         # Use a retry mechanism to check if the health endpoint is available
         for i in {1..5}; do
-          if curl -s http://localhost:8080/healthz | grep "OK"; then
+          if curl -s -o /dev/null -w "%{http_code}" http://localhost:8080/healthz | grep -q "^200$"; then
```

Comment:
nice, i didn't know you could get the exit code that way

Reply:
Really, its in the help for `retry-autopkgtest-regressions`. ;-)